cmd/oras/pull.go

```diff
@@ -122,12 +123,21 @@ func runPull(opts pullOptions) error {
 					s.Annotations = make(map[string]string)
 				}
 				s.Annotations[ocispec.AnnotationTitle] = configPath
-			} else if s.Annotations[ocispec.AnnotationTitle] == "" {
+			}
+			if s.Annotations[ocispec.AnnotationTitle] == "" {
+				// skip fetching unamed leaf nodes
 				ss, err := content.Successors(ctx, fetcher, s)
 				if err != nil {
 					return nil, err
 				}
 				if len(ss) == 0 {
+					if !printed[s.Digest.String()] {
+						err = display.StatusPrinter("Skipped    ", opts.Verbose)(ctx, s)
```

Comment:
You may want to update this with `PrintStatus()`.

Reply:
No, `PrintStatus` implementation will cause `verbose` not to work. Billy and I have checked this scenario.

Reply:
[PrintStatus()](https://github.com/oras-project/oras/blob/9e1d11c51dabb433e904f6c840ded448f8599862/cmd/oras/internal/display/print.go#L52) works with verbose.

If that function does not work, then we need to figure the root cause and fix it.